image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -148,9 +173,22 @@ spec:
       - name: scanner-config-volume
         configMap:
           name: scanner-config
-      - name: scanner-tls-volume
+      {{- if ._rox.env.securedClusterCertRefresh }}
```

Comment:
I think `env` is not the best fit for a setting like this. `env` is supposed to capture properties of the cluster environment (which K8s version, which installation method, is it internet-connected, etc.).

`securedClusterCertRefresh` OTOH is, I would argue, of a different kind, even though it is *influenced* by the installation method (which is also true for many other derived settings).

```
env:
  installMethod: null # string
  openshift: null # bool
  istio: null # bool
  platform: null # string
  offlineMode: null # bool
  proxyConfig: null # string | dict
  storageClasses:
    all: {} # dict
    default: null # string
  pvcs:
    names: null # [string]
```

If I were to suggest an alternative, I would probably go for something like `._rox._securedClusterCertRefresh`. We have this pattern in the Helm chart that fields which are somehow derived internally within the Helm chart are underscore-prefixed. For example:

```
{{- $_ := set $._rox "_scannerV4Enabled" (not $scannerV4Cfg.disable) -}}
```

or
```
{{ $_ = set $._rox "_renderMode" "[< .RenderMode >]" }}
```

WDYT?